src/map.rs

```diff
@@ -1447,6 +1448,7 @@ where
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
We could probably improve these various impls to be:
```rust
impl <K, V, S> FromIterator ... for HashMap<K, V, S>
where
    S: Default
```
That should cover `RandomState`, _and_ work in `no_std`!

Reply:
Yep, good idea!

Reply:
Oh, actually, we may need `epoch::pin()` in some of those places...